### PR TITLE
Add rule to warn against whitespace adjacent to certain fullwidth punctuation characters

### PR DESCRIPTION
**Pull request by wadimw**

### CODE REVIEW

**Comment by wadimw** [comment]
Force-pushed to incorporate declarative rule fixes that were merged to main (#26 #27)

File: src/PluginManager.js [hunk 1]
```diff
@@ -76,7 +76,16 @@ export const regexRules = [
         note: "The full-width characters '{matchString}' are not allowed in the target string. Use ASCII symbols instead.",
         regexps: [ "[\\uFF01|\\uFF05|\\uFF1F]+" ],
         link: "https://github.com/ilib-js/i18nlint/blob/main/docs/resource-no-fullwidth-punctuation-subset.md"
-    }
+    },
+    {
+        type: "resource-target",
+        name: "resource-no-space-with-fullwidth-punctuation",
```
Comment (ehoogerbeets) [inline src/PluginManager.js:82]: Need to add the rule to the "generic" built-in ruleset on line 101

Comment (wadimw) [reply src/PluginManager.js:82]: Fixed in 1d3a920

**Review by ehoogerbeets — APPROVED** [review]
Still need to resolve the merge conflict, but that shouldn't dismiss this approval, so you're good to go.

**Pull request merged**